Makefile

```diff
@@ -90,11 +92,11 @@ lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 
 .PHONY: build
 build: manifests generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/main.go
+	go build -gcflags=${GCFLAGS} -o bin/manager cmd/main.go
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./cmd/main.go
+	go run -gcflags=${GCFLAGS} ./cmd/main.go
```

Comment:
Should we have a separate option for debugging purpose? in case we do want to run with the optimization.

Reply:
that's what happens if you run like this: `GCFLAGS="" make build` or GCFLAGS="" make run`